packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.classNames.ts

```diff
@@ -0,0 +1,50 @@
+import { ITheme, mergeStyleSets } from '../../Styling';
+import { getStyles as getActionButtonStyles } from '../Button/ActionButton/ActionButton.styles';
+import { IButtonClassNames } from '../Button/BaseButton.classNames';
+import { memoizeFunction } from '../../Utilities';
+
+/**
+ * Function to override the getClassNames func in a button.
+ */
+export const getColorPickerGridCellButtonClassNames = memoizeFunction(
+  (
+    theme: ITheme,
+    className: string,
+    variantClassName: string,
```

Comment:
Can you get rid of the parameters that aren't used?

Reply:
Like, have them be underscores ?

Reply:
I wanted to clarify something as I did not understand when I first looked at it first. 
This function to generate classNames is not used by this component. It is passed along its descendant chain to BaseButton so that this can control how BaseButton merges its styles - although this function does things almost exactly like how BaseButton would have done it by default.

I do not have a good explanation of why we needed to have a custom classNames function for the BaseButton - the only thing I could think of was that BaseButton's classNames function was pretty big and it did extra things. But, I also did not want to put more churn into this PR....so I left it like that.